src/app/tests/loginpage.component.spec.ts

```diff
@@ -31,7 +43,38 @@ describe('Loginpage Component', () => {
   });
 
   it('should stub LoginUser method in component', () => {
-    component.LoginUser('', '');
+    component.LoginUser();
     expect(component.message).toEqual('');
   });
+
+  // describe('when service returns an error ', () => {
+  //   let app;
+  //   const testError = {
+  //       status: 406,
+  //       error: {
+  //           message: 'Test 406 error'
+  //       }
+  //   };
+
+  //   beforeEach(async(() => {
+  //     const fixture = TestBed.createComponent(LoginpageComponent);
+  //     app = fixture.componentInstance;
+  //     spyOn(LoginService, 'LoginUser').and.returnValue(Observable.throw(testError));
+  // }));
+  // })
```

Comment:
Should probably delete this comment block

Reply:
done